spring-cloud-config-server/src/main/java/org/springframework/cloud/config/server/environment/vault/SpringVaultClientConfiguration.java

```diff
@@ -125,6 +125,18 @@ public SslConfiguration sslConfiguration() {
 		return new SslConfiguration(keyStoreConfiguration, trustStoreConfiguration);
 	}
 
+	@Override
+	public RestOperations restOperations() {
+		RestTemplate restOperations = (RestTemplate) super.restOperations();
+
+		if (vaultProperties.getNamespace() != null) {
```

Comment:
Let's extract this into a method and also replace https://github.com/spring-cloud/spring-cloud-config/blob/971214645bba8afac71b2e2906a91a36868a0658/spring-cloud-config-server/src/main/java/org/springframework/cloud/config/server/environment/vault/SpringVaultClientConfiguration.java#L98-L102

Reply:
Sure, doing that now. :)